srv/www/regionService/regionInfo.py

```diff
@@ -156,9 +156,14 @@ def create_smt_region_map(conf):
             try:
                 ipaddress.IPv6Network(ip_range)
             except ValueError:
-                msg = 'Could not proces range, improper format: %s'
+                # Besides IPv6 being optional
+                # if IPv6 addresses are present
+                # in /etc/regionService/regionData.cfg file
+                # they must have a valid value
```

Comment:
The comment is superfluous, we are in a code path where we just caught a `ValueError`